### PR TITLE
 RTL styling issues on /firefox/browsers/ page

### PR DESCRIPTION
**Pull request by berliners1**
## Description
I made a small change that will fix this issue, by utilizing z-index to make sure the image is underneath the form container at all times, so the image cannot ever make the form unusable by being in front of it.

## Issue / Bugzilla link
#9110 

## Testing
http://localhost:8000/ar/firefox/browsers/

### CODE REVIEW

**Comment by berliners1** [comment]
Okay, I replaced my z-index-based solution with this one, and to me it looks good now. How does it look for you?

File: media/css/firefox/browsers-products.scss [hunk 1]
```diff
@@ -549,6 +549,10 @@ $url-download-link-hover: svg-url($download-link-hover);
             right: $layout-md;
```
Comment (alexgibson) [inline media/css/firefox/browsers-products.scss:549]: You can remove the existing properties for both `left` and `right` now that the mixin is defining them.

Nit: try and keep things in alphabetical order, so the `@include bidi()` can come first at the top of the list.

Thanks!

**Comment by berliners1** [comment]
Ok, made these changes too. But... I didn't realize @ isn't allowed in commit messages? Oops. Do you know offhand if I can change that?

**Comment by berliners1** [comment]
Ok, my best solution to fixing that was to amend the commit to remove the @, following this documentation: https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message

Let me know if everything looks good with that.

**Pull request merged**